src/DwitTech.AccountService.Core/Dtos/EditRequestDto.cs

```diff
@@ -0,0 +1,61 @@
+﻿using AutoMapper;
+using DwitTech.AccountService.Core.CustomAnnotations;
+using DwitTech.AccountService.Data.Entities;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace DwitTech.AccountService.Core.Dtos
+{
+    [AutoMap(typeof(User), ReverseMap = true)]
```

Comment:
If this is supposed to be a one-way mapping from EditRequestDto to User, then there's no need for reverse mapping.

Reply:
I understand what you mean, possibly you will want me to set the Reverse to false, which makes sense but the implications is that I will only be able to map from entity to dto not the other way round. But I need to map from Dto to an entity.